guides/v2.3/release-notes/backward-incompatible-changes/index.md

```diff
@@ -11,6 +11,58 @@ View a detailed list of PHP code changes that were made in the "2.3-develop" bra
 - [{{site.data.var.ce}} changes]({{ page.baseurl }}/release-notes/backward-incompatible-changes/open-source.html)
 - [{{site.data.var.ee}} changes]({{ page.baseurl }}/release-notes/backward-incompatible-changes/commerce.html)
 
+## API changes
+
+### StoreManager instead of StoreResolve
+
+`\Magento\Store\Api\StoreResolverInterface` was deprecated.
+
+Instead, use `\Magento\Store\Model\StoreManagerInterface`.
+
+When resolving a store for frontend, use `\Magento\Store\Model\StoreManagerInterface::getStore`.
+
+Do not use `\Magento\Store\Api\StoreResolverInterface::getCurrentStoreId`.
+
```

Comment:
A little confused by the second two sentences because there's not a logical transition from the "deprecated" sentence: 

Consider replacing with: 

When resolving for a store frontend, replace occurrences of  `\Magento\Store\Api\StoreResolverInterface::getCurrentStoreId` with `\Magento\Store\Model\StoreManagerInterface::getStore`.